specification/e-referrals-service-api.yaml

```diff
@@ -2392,30 +2392,57 @@ paths:
           $ref: 'components/schemas/responses/TooManyRequests.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
-  /R4/HealthcareService/{id}/_history/{version}:
-    get:
+  /STU3/Appointment/{id}:
+    put:
       security:
         - bearerAuth: []
       description: |
         ## Overview
         To be added....
-      
-      summary: A034 - Retrieve service version
-      operationId: a034-retrieve-service-version
+
```

Comment:
This looks like you have replaced retrieve service version with update appointment

From the spec ERSSUP-8664 retrieve service version should be A037

Reply:
The A034 - Retrieve service version was a placeholder for current version endpoint for services. The Retrieve Service Version endpoint is now moved A037 and It is being developed on another branch. So this place holder is no longer required.